CHANGELOG.md

```diff
@@ -35,6 +39,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 https://tls.mbed.org/tech-updates/releases/mbedtls-2.16.5-and-2.7.14-released)
 release notes for the set of issues addressed.
 
+### Deprecation Notice
```

Comment:
@radhikaj let me know if you think the deprecation notice is fine or need more info/guidance